app/controllers/institutions_controller.rb

```diff
@@ -0,0 +1,16 @@
+class InstitutionsController < ApplicationController
+  def index
+    schools = School
```

Comment:
I can see what you're doing here! Did you consider having two endpoints, one for schools and Local Authorities, and doing the munging in JavaScript? You'd be able to set a `type` on the form and then either do STI, or have distinct `school_id` and `local_authority_id` columns on an application.

Reply:
I don't recall but if I did I probably threw it the window very quickly. accessible autocomplete out the box takes one api endpoint. so you'd have to write some custom javascript to do this munging prior to it hitting the autocomplete code. I don't see a benefit to having two endpoints then combining them when we can just do the same thing server side.